omero/sysadmins/server-upgrade.txt

```diff
@@ -327,15 +328,17 @@ Reset ROI shape color (optional)
 
 Regions of interest generated using OMERO.insight did not follow the
 OME model specification. The color of shapes is now handled according
-to the data model, using RGBA rather than ARGB format.
-To upgrade the color settings of shapes created using OMERO.insight or a client
-saving them in the ARGB format, run the following script:
+to the data model, using RGBA rather than ARGB format. 
+A script is provided to upgrade the color settings of shapes created using OMERO.insight
+or a client saving them in the ARGB format. If you only wish to apply the changes
```

Comment:
maybe "or a different client ..." or "or another client ..." to avoid implicitly contrasting "OMERO.insight" with "a client saving them in the ARGB format"